nussl/ml/networks/modules/blocks.py

```diff
@@ -324,7 +324,7 @@ class Embedding(nn.Module):
         embedding_size (int): Dimensionality of embedding.
         
         activation (list of str): Activation functions to be applied. Options 
-          are 'sigmoid', 'tanh', 'softmax', 'relu'. Unit normalization can be applied by 
+          are 'sigmoid', 'tanh', 'softmax', 'relu', 'gate'. Unit normalization can be applied by 
```

Comment:
I'd rename this to `gated_tanh` to be more descriptive.

Reply:
Done, let's just wait on tests.